thermal/thermal-daemon/thermal-conf.xml

```diff
@@ -1,6 +1,118 @@
 <?xml version="1.0"?>
 <!-- BEGIN -->
 <ThermalConfiguration>
+	<Platform>
+		<Name> TGL NUC i5  </Name>
+		<ProductName> NUC11PAHi5 </ProductName>
```

Comment:
there should not be any space before and after product name.
<ProductName>NUC11PAHi5</ProductName>